backpackerSQL.txt

```diff
@@ -0,0 +1,95 @@
+
+CREATE TABLE Account (
+  AccountId int NOT NULL PRIMARY KEY,

```

Comment:
I would put [an `AUTO_INCREMENT`](https://dev.mysql.com/doc/refman/5.7/en/example-auto-increment.html) on this.